includes/validation/class-amp-validated-url-post-type.php

```diff
@@ -1936,11 +1961,24 @@ public static function print_status_meta_box( $post ) {
 							echo '</p></div>';
 						}
 						?>
-						<?php self::display_invalid_url_validation_error_counts_summary( $post ); ?>
+
+						<?php
+						$counts = self::count_invalid_url_validation_errors( self::get_invalid_url_validation_errors( $post ) );
+
+						if ( 0 < ( $counts['new_rejected'] + $counts['new_accepted'] ) ) {
+							?>
+							<strong id="amp-invalid-markup" class="status-text">
+								<span class="amp-icon amp-warning"></span>
+								<?php esc_html_e( 'Invalid markup not approved', 'amp' ); ?>
+							</strong>
+							<?php
+							esc_html_e( 'Please approve all the invalid markup before updating.', 'amp' );
```

Comment:
This should be iterated on. Something like this:

```suggestion
							esc_html_e( 'Review each instance of invalid markup and determine whether the action of removing (or keeping) is approved. You can change the status and click preview to see what impact the invalid markup has on the page.', 'amp' );
```